src/client/client_metric.h

```diff
@@ -200,8 +201,7 @@ struct MDSClientMetric {
     explicit MDSClientMetric(const std::string& prefix_ = "")
         : prefix(!prefix_.empty()
                      ? prefix_
-                     : "curve_mds_client_" +
-                           std::to_string(reinterpret_cast<uint64_t>(this))),
+                     : "curve_mds_client_" + common::ToHexString(this)),
```

Comment:
why change to hexString? give an example for prefix

Reply:
The prefix in here is just to distinguish different MDSClientMetric's instances because for a specific name, bvar only expose once. So, I just add the instance's address in the prefix, and change the address to hex string is more obvious.